probe/endpoint/conntrack.go

```diff
@@ -0,0 +1,225 @@
+package endpoint
+
+import (
+	"bufio"
+	"encoding/xml"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/weaveworks/scope/test/exec"
```

Comment:
If exec will be used by non-test packages, I guess it shouldn't be nested under test...